cmd/krel/cmd/ff.go

```diff
@@ -62,13 +66,10 @@ as real push if the '--nomock' flag is specified.
 	SilenceUsage:  true,
 	SilenceErrors: true,
 	RunE: func(cmd *cobra.Command, args []string) error {
-		return runFf(ffOpts, rootOpts)
+		return fastforward.RunFf(fastforward.Options)
```

Comment:
We have to set the `nomock` here, then pass the `ffOpts`.
```suggestion
		ffOpts.NoMock = rootOpts.nomock
		return fastforward.RunFf(ffOpts)
```